File: JavaScript/mergestringsalternately.js
const mergeAlternately = function (word1, word2) {
  const n = Math.max(word1.length, word2.length);

  let outPut = "";

  for (let i = 0; i < n; i++) {
    if (i < word1.length) outPut += word1[i];
    if (i < word2.length) outPut += word2[i];
  }
  return outPut;
};

File: Python/MergeStringsAlternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:

        maxLength = max(len(word1), len(word2))
        result = ""

        for i in range(0, maxLength):

            if i < len(word1):
                result += word1[i]
            if i < len(word2):
                result += word2[i]

        return result
    
solution = Solution()


